integration_tests/test-requirements.txt

```diff
@@ -1,4 +1,7 @@
 https://github.com/wazo-platform/wazo-test-helpers/archive/master.zip
+cheroot==8.5.2
+flask==1.1.2
+flask-restful==0.3.8
```

Comment:
as mentionned, preferably split out these requirements into a separate file / a direct pip invocation.
The `test-requirements.txt` file should be exclusively for the "test runtime"(what is run by `pytest`) and not any mock under test.